start.d.ts

```diff
@@ -0,0 +1,2 @@
+import agent from './';
```

Comment:
Why you need this file?

Reply:
Because there's two ways to require the agent:

```js
const agent = require('elastic-apm-node')
```

And:

```js
const agent = require('elastic-apm-node/start')
```

So I think that's the only way to do it right?

Reply:
Ok, I didn't know that!

> So I think that's the only way to do it right?

The only way to be sure, I guess, is to try it with VSCode or the tsc compiler.

Reply:
I tried to require the both versions in VS Code and it seemed to work fine. So I think it's ok. I just wasn't sure if there were a better way to do it or if my way was ok